modules/teadsBidAdapter.js

```diff
@@ -1,14 +1,17 @@
 import {registerBidder} from '../src/adapters/bidderFactory.js';
-const utils = require('../src/utils.js');
+import {getStorageManager} from '../src/storageManager.js';
+import * as utils from '../src/utils.js';
+
 const BIDDER_CODE = 'teads';
 const GVL_ID = 132;
-const ENDPOINT_URL = 'https://a.teads.tv/hb/bid-request';
+const ENDPOINT_URL = '//SSP_PORT_8080_TCP_ADDR:SSP_PORT_8080_TCP_PORT/hb/bid-request';
```

Comment:
What exactly is going on here?

This looks like some macro to be replaced but I do not see it happening anywhere in your adapter?

`SSP_PORT_8080_TCP_ADDR:SSP_PORT_8080_TCP_PORT`

Reply:
yes it's an error on our side, we will revert that change!